### PR TITLE
2379 - adding some ux feedback to fix text and icons

### PR DESCRIPTION
**Pull request by codyseibert**
- show different icon depending on if the added docket numbers was set or not
- changing the modal text to be static

### CODE REVIEW

File: web-client/src/presenter/computeds/createOrderHelper.js [hunk 1]
```diff
@@ -22,9 +22,10 @@ export const createOrderHelper = (get, applicationContext) => {
   const isLeadCase = caseDetail.leadDocketNumber === caseDetail.docketNumber;
 
   return {
+    addDocketNumbersButtonIcon: addedDocketNumbers ? 'edit' : 'plus-circle',
     addDocketNumbersButtonText: addedDocketNumbers
-      ? 'Edit docket numbers to the coversheet'
-      : 'Add docket numbers to the coversheet',
+      ? 'Edit docket numbers in the cover sheet'
```
Comment (TomElliottFlexion) [inline web-client/src/presenter/computeds/createOrderHelper.js:27]: Nit: 
```suggestion
      ? 'Edit docket numbers on the cover sheet'
```

File: web-client/src/presenter/computeds/createOrderHelper.test.js [hunk 2]
```diff
@@ -87,7 +88,8 @@ describe('createOrderHelper', () => {
     });
 
     expect(result.addDocketNumbersButtonText).toEqual(
-      'Edit docket numbers to the coversheet',
+      'Edit docket numbers in the cover sheet',
```
Comment (TomElliottFlexion) [inline web-client/src/presenter/computeds/createOrderHelper.test.js:91]:
```suggestion
      'Edit docket numbers on the cover sheet',
```

**Review by TomElliottFlexion — APPROVED** [review]

File: web-client/src/presenter/computeds/createOrderHelper.js (hunk 1 above)
Comment (codyseibert) [reply web-client/src/presenter/computeds/createOrderHelper.js:27]: ux asked for `to` and `in` specifically

File: web-client/src/presenter/computeds/createOrderHelper.test.js (hunk 2 above)
Comment (codyseibert) [reply web-client/src/presenter/computeds/createOrderHelper.test.js:91]: same as above, ux asked for `to` and `in` specifically

File: web-client/src/presenter/computeds/createOrderHelper.js (hunk 1 above)
Comment (TomElliottFlexion) [reply web-client/src/presenter/computeds/createOrderHelper.js:27]: Obviously I'll defer, but `on` would be correct

**Pull request merged**